strings/facts.txt

```diff
@@ -0,0 +1,26 @@
+The five most populous species within the Trans-Solar Federation are Humans, Skrell, Kidan, Slime People, and Unathi, comprising 92% of the Federation's population collectively.
+A belt of shipwrecks, asteroids, and space debris surrounds Epsilon Eridani II. This belt's origins are unknown, and new additions seem to arrive from nowhere.
+The Union of Sovereign Socialist Planets, better known as the USSP, is an alliance of fourteen nominally independent socialist and left-wing nations in the upper Orion Arm. Comprising a powerful economic and military bloc with great regional influence, they consider themselves the primary rival to the Trans-Solar Federation.
+The vast majority of vox in Orion are found aboard moon-sized Arkships traveling the void between stars. What tales that emerge from Vox able to escape them typically dissuades any explorers from seeking out an Ark.
+The ocean planet of Xarxis is the homeworld of the gelatinous Slime People. Once subjected to devastating seasonal temperature fluctuations, a joint project by the Trans-Solar Federation and Xarxis governments constructed a series of orbital mirrors to stabilize the world's climate.
+Epsilon Eridani Regional Headquarters, better known as Central Command, serves as Nanotrasen's de-facto capital for the Epsilon Eridani system, and is responsible for directly overseeing the major research stations in orbit of Epsilon Eridani II.
+Centuries ago, the many warring nations of the Kidan were united as a single Empire under the psychic dominion of an immensely powerful creature known as the Empress. This control was ended at the end of the Kidan War, the largest conflict in the Trans-Solar Federation's history.
+Though Nanotrasen is best known for its hold on the plasma fuel market, the company holds stakes in numerous other fields, from consumer electronics to energy weapon systems.
+The Free Eridani Republic is a small but economically powerful nation in the heart of the Orion sector. One of the sector's most corporate-friendly nations, Eridani is home to a host of powerful companies, most notably Nanotrasen.
+Once the seat of the Plasma Revolution and a bustling center of Federation commerce, Mars has been in the grip of economic depression since the late 2300s, as its plasma reserves ran dry and tourism efforts proved untenable. Though many have written off the Red Planet, recent investments into mining and heavy industry may signal a turnaround in the world's fortunes.
+Persistent rumors abound regarding Nanotrasen's implementation of "contingency plans" within their plasma refining infrastructure as a safeguard against nationalization or attack. Nanotrasen considers all such rumors to be baseless slander.
+The Heavenly State of Lei-Gong is a small, highly religious democracy in the Western Orion region. Predominently Buddhist and steadfastly neutral, Lei-Gong is a bastion of peace in the otherwise chaotic Western Approach.
```

Comment:
```suggestion
The Heavenly State of Lei-Gong is a small, highly religious democracy in the Western Orion region. Predominantly Buddhist and steadfastly neutral, Lei-Gong is a bastion of peace in the otherwise chaotic Western Approach.
```